include/aikido/planner/vectorfield/detail/VectorFieldPlannerExceptions.hpp

```diff
@@ -0,0 +1,81 @@
+#ifndef AIKIDO_PLANNER_VECTORFIELD_VECTORFIELDPLANNEREXCEPTIONS_HPP_
+#define AIKIDO_PLANNER_VECTORFIELD_VECTORFIELDPLANNEREXCEPTIONS_HPP_
+
+#include <stdexcept>
+#include <string>
+#include <dart/dynamics/dynamics.hpp>
+
+namespace aikido {
+namespace planner {
+namespace vectorfield {
+
+/// Define termination of vectorfield planner.
+// When VectorFieldTerminated is throws, it means
+// that planning is finished.
+class VectorFieldTerminated : public std::exception
```

Comment:
This is an implementation detail of `followVectorField()` used to terminate the integrator. As such, we should take this out of the header file and put it in the `.cpp` file where it is used in an anonymous `namespace.`